website/core/Footer.js

```diff
@@ -49,6 +46,19 @@ class Blast extends React.Component {
   color: white;
   text-decoration: underline;
 }
+
+body {
+  // let's give it a minimal animation
+  transition: ease 1.5s;
```

Comment:
this adds a tiny animation to ease in the top at the load of the page. Not sure we want to have this, considering that one can't "x" it away and it moves the content down by 50px on every page load. Might want to switch it off, even if that means there is a short flicker for re-render on slower/delayed loads of the footer.